src/pages/LogOutPreviousUserPage.js

```diff
@@ -13,7 +13,13 @@ const propTypes = {
     session: PropTypes.shape({
         /** The user's email for the current session */
         email: PropTypes.string,
-    }).isRequired,
+    }),
+};
+
+const defaultProps = {
+    session: {
+        email: null,
+    },
```

Comment:
We can just set the empty object to this because we use `lodashGet` to get the email, and it has a default value empty string.

`const sessionEmail = lodashGet(this.props.session, 'email', '');`

Reply:
Yeah, we can do that. Could you check if the empty string passed differs from the `null` default value?